File: 2023/python/daily-template/src/part1.py
from dotenv import load_dotenv
import os

load_dotenv()
filename = os.getenv('TEXTS_PATH')+'{{day}}.txt'
with open(filename, 'r') as file:
    data = [line.strip() for line in file]


def part1(data: list[str]) -> int:
    result = 0
    for line in data:
        pass
    return result


if __name__ == "__main__":
    print(part1(data))

File: 2023/rust/daily-template/src/bin/part1.rs
use {{crate_name}}::part1::process;
use miette::Context;

fn main() -> miette::Result<()> {
    let file = include_str!("../../../../day-01.txt");
    let result = process(file).context("process part 1")?;
    println!("{}", result);
    Ok(())
}